docs/cave/d2d-gis-shapefiles.md

```diff
@@ -5,10 +5,12 @@ This section describes how to start the GIS application, configure the display a
 
 ## Display GIS Data
 
-The GIS application is accessed through the CAVE menu by first selecting the Import option, and then choosing GIS Data from the Import submenu. The GIS DataStore Parameters dialog is comprised of four sections:
+The GIS application is accessed through **File > Import > GIS Data**. The GIS DataStore Parameters dialog is comprised of four sections:
```

Comment:
I would remove all instances of the phrase "GIS application" and replace it with something more appropriate.
In this specific instance I think it could be "Importing a GIS file is accessed through **File > Import > GIS Data**" or maybe "The GIS Import dialog is accessed through **File > Import > GIS Data**".